examples/Bunch/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -128,25 +128,9 @@ using PIC_Ions = Particles<
 
 /*########################### end species ####################################*/
 
-
-/*! we delete this ugly definition of VectorAllSpecies after all picongpu components
- * support multi species */
-/** \todo: not nice, but this should be changed in the future*/
-using Species1 = MakeSeq_t<
-#if( ENABLE_ELECTRONS == 1 )
-    PIC_Electrons
-#endif
->;
-
-using Species2 = MakeSeq_t<
-#if( ENABLE_IONS == 1 )
-    PIC_Ions
-#endif
->;
-
 using VectorAllSpecies = MakeSeq_t<
-    Species1,
-    Species2
+    PIC_Electrons,
+    PIC_Ions
```

Comment:
REQUIRED:
There are no ions in this example `ENABLE_IONS ` was `0` thus `Species2` was empty.